P5/Source/Specs/charProp.xml

```diff
@@ -26,20 +28,28 @@ $Id$
     <memberOf key="att.typed"/>
   </classes>
   <content>
-    <!-- ((unicodeName|localName), value) -->
     <sequence>
       <alternate>
         <elementRef key="unicodeName"/>
         <elementRef key="localName"/>
       </alternate>
       <elementRef key="value"/>
     </sequence>
-    <!--s:assert> some schematron rules here</s:assert-->
   </content>
+  <!-- TODO Phase 3: Mention conversion script as part of deprecation warning. #1805 -->
+  <constraintSpec scheme="schematron" validUntil="2021-07-09"
```

Comment:
Delete the entire `<constraintSpec>`. The deprecation Schematron check will (or at least should) be automatically generated at build time because of the `@validUntil` on the `<elementSpec>`.